airbyte-webapp/src/components/base/SecretTextArea/SecretTextArea.module.scss

```diff
@@ -0,0 +1,21 @@
+@use "../../../scss/colors";
+
+button.toggleVisibilityButton {
```

Comment:
Since we usually have classNames only, is there a specific reason we're going with `button.toggleVisibilityButton` here?

Reply:
No good reason. I may have done this during the early commits and forgot to remove it.